charts/DataTab.tsx

```diff
@@ -1,11 +1,12 @@
-import { flatten, uniq, sortBy, extend, csvEscape } from "./Util"
+import { flatten, uniq, sortBy, extend, csvEscape, last, first } from "./Util"
 import { Bounds } from "./Bounds"
 import * as React from "react"
 import { computed, action } from "mobx"
 import { observer } from "mobx-react"
 import { ChartConfig } from "./ChartConfig"
 import { faDownload } from "@fortawesome/free-solid-svg-icons/faDownload"
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome"
+import { DimensionWithData } from "./DimensionWithData"
```

Comment:
Note: you might want to rebase onto master. I pushed a change yesterday where I started moving the data classes and interfaces to a subfolder. `DimensionWithData` is now temporarily `ChartDimensionWithOwidVariable` to be more clear specifically what it is referring to. I'm trying to reduce and standardize terminology across the codebase hoping that might make somethings easier (and hopefully I'm not creating merge conflicts to just shuffle bits around).

Reply:
Got it, will fix.
Do you think it is necessary to specify "Owid" variable?

Reply:
> Do you think it is necessary to specify "Owid" variable?

Maybe? Not sure. Sometimes when in doubt about these things I pick the more obscure word that will be easier to do a global rename (easier to rename all OwidVariables to Variables than vice versa). What I like about OwidVariable is that it maps to something very specific in our domain (maps to a database table, etc) whereas the general term "Variable" is heavily overloaded.

Reply:
That makes sense.